docs/generated/workloads.md

```diff
@@ -2068,35 +2068,6 @@ Command synchronously establishes temporary connections between cluster nodes.
 
 
 
-## [AggregateExpressions](https://www.github.com/mongodb/genny/blob/master/src/workloads/query/AggregateExpressions.yml)
-### Owner
-Query Execution
```

Comment:
Should we also delete the following reference: https://github.com/mongodb/genny/blob/7401c77020d0911ffbeb8c77182353416ef89ee2/docs/generated/phases.md#aggregateexpressions ? I guess the thinking is 'no' because while we are disabling the workload, we are not deleting the phase because it is referenced elsewhere ([for instance](https://github.com/mongodb/genny/blob/7401c77020d0911ffbeb8c77182353416ef89ee2/src/workloads/query/CumulativeWindows.yml#L72)), right?